### DISCUSSION TITLE
Looking for a solution to clear the existing cattle id

### DISCUSSION THREAD

**Discussion by DipindasThayyil** [rancher/rke2#7141] [Q&A]
While re-registering the node, I'm getting below error. Is there any fix for this? 

 Cattle ID was already detected as d940872a74145fd9f197c2e212542c31bb15e92aa2ac380cc2964c934a2c707. Not generating a new one

Rancher | v2.8.5

**Comment by brandond** [comment]
Is that an error? I don't think that's an error. It's just telling you that there are files left over from a previous install and it is reusing the ID.

**Reply by DipindasThayyil** [reply]
How can I remove that? cleaned up node using rke clean up script available on the /usr/bin/local path and trying to re-add it now.

**Reply by brandond** [reply]
This is a Rancher question, not RKE2.

**Reply by DipindasThayyil** [reply]
I'm getting the above error from RKE2 only.

**Reply by brandond** [reply]
You're getting the error from a Rancher install script. RKE2 does not have or use a "Cattle ID" anywhere internally.
